qp_shotgun/humann2/tests/test_humann2.py

```diff
@@ -146,8 +153,8 @@ def test_generate_humann2_analysis_commands_forward_reverse(self):
             '--xipe "off" --annotation-gene-index "8" '
             '--protein-database "uniref" --threads "1" --pathways "metacyc" '
             '--pick-frames "off" --translated-alignment "diamond" '
-            '--log-level "DEBUG"',
-            'humann2 --input "fastq/s1.R2.fastq" --output "output/s1.R2" '
+            '--remove-column-description-output --log-level "DEBUG"' % out_dir,
```

Comment:
Is the % out_dir on the wrong line here? should it go after `--output "%s/s1.R2" '` on line 157?

...and on line 143,172, 185, 200, 214, and 227?